buildlogger/client.go

```diff
@@ -0,0 +1,279 @@
+package buildlogger
```

Comment:
package name should match repo name

Reply:
the back is buildlogger -> timber/buildlogger